client/lib/mixins/i18n/index.js

```diff
@@ -14,7 +14,7 @@ var debug = require( 'debug' )( 'calypso:i18n' ),
 var config = require( 'config' ),
 	numberFormatPHPJS = require( './number-format' ),
 	emitter = require( 'lib/mixins/emitter' ),
-	interpolateComponents = require( 'lib/interpolate-components' );
+	interpolateComponents = require( 'lib/interpolate-components/src' );
```

Comment:
I'm pretty sure webpack reads `package.json` `main` property in subfolders. We already have a few of those in `wp-calypso`, check `client/my-sites/sidebar` for instance ;)